docs/offscreen-canvas-support.md

```diff
@@ -0,0 +1,250 @@
+# OffscreenCanvas Support in Vega
```

Comment:
There is a lot of redundancy with other pages like the examples.md files. Can we consolidate?